docs/root/configuration/upstream/cluster_manager/cluster_runtime.rst

```diff
@@ -143,6 +143,11 @@ upstream.use_http2
   Whether the cluster utilizes the *http2* if configured in `HttpProtocolOptions <envoy_v3_msg_config.upstreams.http.v3.HttpProtocolOptions>`.
   Set to 0 to disable HTTP/2 even if the feature is configured. Defaults to enabled.
 
+upstream.use_http3
+  Whether the cluster utilizes the *http3* if configured in `HttpProtocolOptions <envoy_v3_msg_config.upstreams.http.v3.HttpProtocolOptions>`.
```

Comment:
It looks like this text is identical to the HTTP/2 text (except 3 instead of 2), but I wonder if we should remove "the"  after "utilizes"?  "Whether the cluster utilizes *http3* if configured..." WDYT?